### PR TITLE
Consistency in Application Connectivity docs 

### LINKED ISSUE #12714: Consistent naming and explanation of the < NAME_OF_YOUR_APP > variable
**Description**

The variable `NAME_OF_YOUR_APP` used in the tutorial series:
https://kyma-project.io/docs/kyma/latest/03-tutorials/00-application-connectivity/

Should be consistent and an explanation of that variable in the first tuorial step would be great.
This is the name of the "external application you want to consume" would also be great in the guide. Choose a descriptive name here or something similar.

**Area**
https://kyma-project.io/docs/kyma/latest/03-tutorials/00-application-connectivity/ac-03-application-mapping/
In the screenshot you can see that the variable is not consistant. 
Screenshot 2021-11-25 at 16 57 40

### PR DESCRIPTION
**Pull request by VOID404**
**Description**
- Make insecure certificates warning more consistent
- Remove some weird spaces
- Move certificate warnings before the step where the problem occurs (or into prerequisites, where they exist)
- Review the docs to resolve #12714 

**Related issue(s)**
#12714

### CODE REVIEW

File: docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md [hunk 1]
```diff
@@ -12,40 +12,25 @@ Follow these steps to detect the source of the problem:
...
 1. Check the logs.
 
-    Check the logs from the Application Gateway Pod to verify that the call reached Application Gateway.
-    To fetch these logs, run this command:
+Check the logs from the Application Gateway Pod to verify that the call reached Application Gateway.
+To fetch these logs, run this command:
+
+```bash
+kubectl -n kyma-system logs -l app=central-application-gateway -c central-application-gateway
+```
```
Comment (majakurcius) [inline docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md:20]:
``````suggestion
   Check the logs from the Application Gateway Pod to verify that the call reached Application Gateway.
   To fetch these logs, run this command:

   ```bash
   kubectl -n kyma-system logs -l app=central-application-gateway -c central-application-gateway
   ```
``````
These are supposed to be indented, as the code block belongs to step 1.

File: docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md [hunk 2]
```diff
@@ -12,40 +12,25 @@ Follow these steps to detect the source of the problem:
...
     
-    ```bash
-    kubectl -n kyma-system logs -l app=central-application-gateway -c central-application-gateway
-    ```
-   
-    The request that reached the Pod is logged by Application Gateway.
+The request that reached the Pod is logged by Application Gateway.
```
Comment (majakurcius) [inline docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md:22]:
```suggestion
   The request that reached the Pod is logged by Application Gateway.
```

File: docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md [hunk 3]
```diff
@@ -12,40 +12,25 @@ Follow these steps to detect the source of the problem:
...
+To check the target URL of the API, fetch the Service definition from Application Registry:
+
+```bash
+curl https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services/{SERVICE_ID} --cert {CERTIFICATE_FILE} --key {KEY_FILE}
+```
+
+> **CAUTION:** On a local Kyma deployment, skip SSL certificate verification when making a `curl` call, by adding the `-k` flag to it. Alternatively, add the Kyma certificates to your local certificate storage on your machine using the `kyma import certs` command.
+
+A successful call returns a `json` response with the service definition that contains the target URL.
+Call the target URL directly to verify that the value of **api.targetUrl** is correct.
```
Comment (majakurcius) [inline docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md:36]:
``````suggestion
   If your call reaches Application Gateway, but you still receive an error, check if the API URL in the service definition matches the API URL of the actual service you are trying to call.
   To check the target URL of the API, fetch the Service definition from Application Registry:

   > **CAUTION:** On a local Kyma deployment, skip SSL certificate verification when making a `curl` call, by adding the `-k` flag to it. Alternatively, add the Kyma certificates to your local certificate storage on your machine using the `kyma import certs` command.

   ```bash
   curl https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services/{SERVICE_ID} --cert {CERTIFICATE_FILE} --key {KEY_FILE}
   ```

   A successful call returns a `json` response with the service definition that contains the target URL.
   Call the target URL directly to verify that the value of **api.targetUrl** is correct.
``````

The info in the note is something that the user should know before they make the call, so it should precede the command.

**Review by majakurcius — COMMENTED** [review]
The indentation should be left intact as it ensures that content belonging to a particular step is displayed as its part.

File: docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md (hunk 3 above)
Comment (VOID404) [reply docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md]: That makes it inconsistent with the rest of the docs (eg [Revoke a client certificate](https://github.com/kyma-project/kyma/blob/main/docs/03-tutorials/00-application-connectivity/ac-07-revoke-client-cert.md))

Comment (majakurcius) [reply docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md]: Oh, touché... You're right. Let's make it consistent. 🚀

**Review by majakurcius — APPROVED** [review]

**Pull request merged**